app/Place/PlaceControllerProvider.php

```diff
@@ -76,19 +73,10 @@ function (Application $app) {
                 $app['place_repository'],
                 $app['uuid_generator'],
                 new PlaceDenormalizer(),
-                new CombinedRequestBodyParser(
-                    new ImportLabelVisibilityRequestBodyParser(
-                        $app[LabelServiceProvider::JSON_READ_REPOSITORY],
-                        $app[LabelServiceProvider::RELATIONS_READ_REPOSITORY]
-                    ),
-                    new ImportTermRequestBodyParser(
-                        new PlaceCategoryResolver()
-                    )
-                ),
+                new ImportTermRequestBodyParser(new PlaceCategoryResolver()),
```

Comment:
Just a suggestion but maybe keep it wrapped in `CombinedRequestBodyParser` so the next person who needs to add a `RequestBodyParser` here knows it exists and just adds the extra `RequestBodyParser` instead of e.g. adding an extra argument to the constructor for it. I don't think the decoration by `CombinedRequestBodyParser` will have a big performance hit.

Reply:
The `CombinedRequestBodyParser` is used again. This was also needed after merging aster and the changes for the `ImportPriceInfoRequestBodyParser`